migrate/20251202_add_limited_reputation.rb

```diff
@@ -0,0 +1,58 @@
+# frozen_string_literal: true
+
+Sequel.migration do
+  up do
+    # Add temporary VARCHAR column
+    alter_table(:project) do
+      add_column :reputation_new, String, collate: '"C"', default: "new"
```

Comment:
Instead of add/drop/rename column, you can change the column type, update the default, and  add the check constraint:

```suggestion
      set_column_type :reputation, "text collate \"C\"", using: Sequel.cast(:t, String)
      set_column_default :reputation, "new"
      add_constraint :reputation_check, reputation: %w[new verified limited]
```

Currently, you need the `"text collate \"C\""` because Sequel doesn't recognize the `:collate` option for `set_column_type`. I'll fix that.

Reply:
I updated the migration